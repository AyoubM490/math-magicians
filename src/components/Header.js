import React from 'react';

const Header = () => (
  <div>
    <h1 className="bigHeading">Math Magicians</h1>
  </div>
);

export default Header;
