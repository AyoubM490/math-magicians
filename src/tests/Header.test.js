import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('test if Header', () => {
  test('renders Header correctly', () => {
    const component = renderer.create(<Header />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
