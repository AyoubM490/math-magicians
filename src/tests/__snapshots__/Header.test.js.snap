// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if Header renders Header correctly 1`] = `
<div>
  <h1
    className="bigHeading"
  >
    Math Magicians
  </h1>
</div>
`;
